### PR TITLE
[DOCS] Add PR list for 7.0.0-alpha2

### PR DESCRIPTION
**Pull request by tomcallahan**
Add PR list from ES for 7.0.0-alpha2

### CODE REVIEW

File: docs/reference/release-notes/7.0.0-alpha2.asciidoc [hunk 1]
```diff
@@ -1,10 +1,599 @@
...
+* Add Tests for findSamlRealm {pull}35905[#35905]
+* Add realm information for Authenticate API {pull}35648[#35648]
+* Formal support for "password_hash" in Put User {pull}35242[#35242] (issue: {issue}34729[#34729])
+
+Authorization::
+* Improve exact index matching performance {pull}36017[#36017]
+* `manage_token` privilege for `kibana_system` {pull}35751[#35751]
+* Grant .tasks access to kibana_system role {pull}35573[#35573]
+
+Build::
```
Comment (lcawl) [inline docs/reference/release-notes/7.0.0-alpha2.asciidoc]: We used to omit PRs with "Build" labels.  Should these Infra/Build PRs be omitted too?

File: docs/reference/release-notes/7.0.0-alpha2.asciidoc [hunk 2]
```diff
@@ -1,10 +1,599 @@
...
+
+Audit::
+* Fix origin.type for connection_* events {pull}36410[#36410]
+* Fix IndexAuditTrail rolling restart on rollover edge {pull}35988[#35988] (issue: {issue}33867[#33867])
+
+Authentication::
+* Fix kerberos setting registration {pull}35986[#35986] (issues: {issue}30241[#30241], {issue}35942[#35942])
+* Add support for Kerberos V5 Oid {pull}35764[#35764] (issue: {issue}34763[#34763])
+
+Build::
```
Comment (lcawl) [inline docs/reference/release-notes/7.0.0-alpha2.asciidoc]: Ditto "Infra/Build" omission

File: docs/reference/release-notes/7.0.0-alpha2.asciidoc [hunk 3]
```diff
@@ -1,10 +1,599 @@
...
+Java Low Level REST Client::
+* Make warning behavior pluggable per request {pull}36345[#36345]
+* Add PreferHasAttributeNodeSelector {pull}36005[#36005]
+
+Geo::
+* Integrate Lucene's LatLonShape (BKD Backed GeoShapes) as default `geo_shape` indexing approach {pull}36751[#36751] (issue: {issue}35320[#35320])
+* Integrate Lucene's LatLonShape (BKD Backed GeoShapes) as default `geo_shape` indexing approach {pull}35320[#35320] (issue: {issue}32039[#32039])
+
+Machine Learning::
+* [ML-DataFrame] move common fields and string into a utility class {pull}36557[#36557]
```
Comment (lcawl) [inline docs/reference/release-notes/7.0.0-alpha2.asciidoc]: @hendrikmuhs Do we want all these dataframe-related PRs in the release notes?

Comment (hendrikmuhs) [reply docs/reference/release-notes/7.0.0-alpha2.asciidoc]: no, it's not in, it's on a feature branch. Sorry for the mess. I will remove the labels on all those PR's

TL;DR;

We want to ship this feature in 7.0 if everything works out, having that said I think these individual changes are to much detail for release notes. Once we approach branch landing we should revisit how we announce it in release notes

Thanks for bringing it up @lcawl and again, sorry for the mess

Comment (lcawl) [reply docs/reference/release-notes/7.0.0-alpha2.asciidoc]: Thanks @hendrikmuhs I removed all those dataframe PRs from this list
